decidim-core/app/views/layouts/decidim/mailer.html.erb

```diff
@@ -126,6 +125,22 @@
           </center>
         </td>
       </tr>
+      <tr>
+        <td class="float-center" align="center" valign="top">
+          <center>
+            <table class="container">
+            <tr>
```

Comment:
Can you still fix the indentation on this `<tr>`? It should be one level deeper, on the same level as the `<tr>` below it.